content/hi/contributor-ladder/_index.md

```diff
@@ -0,0 +1,100 @@
+
+## 2. अप्रूवर (Approvers) 
+
+अप्रूवर या अनुमोदनकर्ता, PR (पुल अनुरोध) पर प्रतिक्रिया प्रदान करते हैं और उन्हें अनुमोदित करते हैं। कोई भी सक्रिय योगदानकर्ता अप्रूवर बन सकता है (देखें "अप्रूवर बनना")। शब्दावली दो अप्रूवर के बीच अंतर करती है: (1) अंग्रेजी शब्दावली के लिए अप्रूवर और (2) स्थानीयकरण टीमों के लिए अप्रूवर 
+
+शब्दावली अनुमोदनकर्ताओं से अपेक्षा की जाती है कि वे:
+
+- तकनीकी सटीकता के लिए PR की समीक्षा करें,
+- योगदानकर्ताओं के मुद्दों को नियुक्त करें और उन्हें उचित रूप से अंकित करें,
+- योगदानकर्ताओं को फीडबैक प्रदान करें और जरूरत पड़ने पर उनका मार्गदर्शन करें,
+- पुल अनुरोध को प्रूफरीड और एडिट करें।
+
+यदि कोई अप्रूवर अब उपरोक्त कर्तव्यों में दिलचस्पी नहीं रखता है या नहीं कर सकता है, तो उन्हें मेंटेनर को बताना चाहिए और पद छोड़ देना चाहिए।
+
+### अंग्रेजी शब्दावली स्वीकृतकर्ता
+
+तीन प्रकार के अप्रूवर हैं:
+
+1) मजबूत तकनीकी पृष्ठभूमि वाले अप्रूवर,
+2) ठोस लेखन कौशल वाले अप्रूवर,
+3) अप्रूवर जो दोनों में दक्ष हैं।
+
+**तकनीकी  अप्रूवर**: एक मजबूत तकनीकी पृष्ठभूमि वाले व्यक्ति, ठोस अंग्रेजी लेखन कौशल के बिना भी अनुमोदनकर्ता हो सकते हैं। हालांकि, अगर वे तकनीकी योग्यता के आधार पर PR को मंजूरी देते हैं, तो उन्हें यह सुनिश्चित करना होगा कि इसकी समीक्षा एक संपादक (अप्रूवर) द्वारा की जाए।
+
+**संपादक**: संपादक शब्दों का प्रूफरीड करते हैं और सुनिश्चित करते हैं कि उन्हें शैली मार्गदर्शिका के अनुसार सरल भाषा में समझाया गया है। यदि कोई शब्द भारी रूप से संपादित किया गया है, तो संपादक को तकनीकी अप्रूवर से अनुरोध करना चाहिए कि वह इसकी फिर से समीक्षा करे ताकि यह सुनिश्चित हो सके कि अर्थ में कोई बदलाव नहीं किया गया है।
+
+### स्थानीयकरण अप्रूवर
+
+शब्दावली में स्थानीयकरण अप्रूवर भी हैं। ये स्थानीयकरण टीमों (शब्दावली का अनुवाद करने वाली टीमों) में से एक के लिए अप्रूवर हैं। स्थानीयकरण अप्रूवर को केवल अपनी टीम के लिए अप्रूवर कर्तव्यों का पालन करने की अनुमति है और उनके पास पुल अनुरोध को अपनी समर्पित डेवलपमेंट ब्रांच में विलय करने की क्षमता है। कोई भी स्थानीयकरण अप्रूवर अंग्रेजी शब्दावली का अप्रूवर भी बन सकता है यदि वे आवश्यकताओं को पूरा करते हैं।
+
+### एक अप्रूवर बनना 
+
+स्वीकृति देने वाले उम्मीदवारों के पास उच्च गुणवत्ता वाले पुल अनुरोध जमा करने का एक सिद्ध अभिलेख होना चाहिए और दूसरों को अपने पुल अनुरोध को विलय योग्य स्थिति में लाने में मदद करना चाहिए। यदि उनका समय क्षेत्र अनुमति देता है, तो उन्हें नियमित रूप से [शब्दावली कार्य समूह की बैठकों] (https://www.cncf.io/calendar/) में भी भाग लेना चाहिए।
+
+अप्रूवर बनने के लिए, मौजूदा मेन्टेनेरों के प्रति रुचि व्यक्त करके प्रारंभ करें। इसके बाद मौजूदा मेन्टेनेर आपसे पुल अनुरोध में योगदान करके, समीक्षा करके, और उनके मार्गदर्शन में ऐसे अन्य कार्यों को करके उपरोक्त योग्यताओं को प्रदर्शित करने के लिए कहेंगे। एक साथ काम करने के कुछ समय बाद, मेन्टेनेर्स यह तय करेंगे कि आपको अप्रूवर का दर्जा देना है या नहीं। यह निर्णय आपकी दक्षता और जवाबदेही के प्रदर्शित स्तर पर आधारित होगा।
+
+## 3. मेन्टेनेर्स
+
+मेन्टेनेर्स वे अप्रूवर होते हैं जो पुल अनुरोध का विलय भी कर सकते हैं। कोई भी योगदानकर्ता शब्दावली मेन्टेनर बन सकता है (देखें "एक मेन्टेनर बनना")। मेन्टेनेरों के लिए कुछ अपेक्षाएँ हैं, जिनमें शामिल हैं:
```

Comment:
I think the spelling of maintainers will be मेन्टेनर्स

Reply:
We've established maintainer as मेन्टेनर so मेन्टेनेरों should be easily understandable for the reader

Reply:
done